demos/JWST/S3_nircam_wfss_template.ecf

```diff
@@ -12,11 +12,16 @@ ywindow         [5,64]      # Vertical axis as seen in DS9
 xwindow         [100,1700]  # Horizontal axis as seen in DS9
 src_pos_type    gaussian    # Determine source position when not given in header (Options: header, gaussian, weighted, max, or hst)
 record_ypos     True        # Option to record the y position and width for each integration (only records if src_pos_type is gaussian)
+dqmask          False       # Mask pixels with an odd entry in the DQ array
```

Comment:
Shouldn't this be set to True?

Reply:
No, NIRCam seems to perform better when `dqmask` is set to False.

Reply:
Huh, surprising but good to know